docs/BasicCommunication/OnAppPropertiesChange/assets/OnAppPropertiesChange.txt

```diff
@@ -0,0 +1,19 @@
+title OnAppPropertiesChange 
+
+participant App
+participant SDL
+participant HMI
+
+==<align:center>Pre-conditions: 1. SDL and HMI are started==  
```

Comment:
i think these preconditions can be assumed :)

Reply:
@iCollin, do you want me to remove them?

Reply:
Yes please remove them.

Reply:
@iCollin 
updated in 69f87d9f974014fc0a11c6185c696554c4becbaa